tests/ssl-opt.sh

```diff
@@ -8889,23 +8873,126 @@ run_test    "TLS1.3: minimal feature sets - gnutls" \
             -c "<= parse finished message" \
             -c "HTTP/1.0 200 OK"
 
+requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL
+skip_handshake_stage_check
+requires_gnutls_tls1_3
+run_test    "TLS1.3:Not supported version check:gnutls: srv max TLS 1.0" \
```

Comment:
```suggestion
run_test    "TLS 1.3: Not supported version check:gnutls: srv max TLS 1.0" \
```
To align with the prefix of the generated tests. Please fix them all in this file. That way we can easily run all TLS 1.3 and only TLS 1.3 tests with: `./tests/ss-opt.sh -f "TLS 1.3".